script.js

```diff
@@ -0,0 +1,11 @@
+isThisMyName('Adam');
+
+
+function isThisMyName(firstName) {
+    if (firstName === "Natasza") {
+        result = true;
+    } else {
+        result = false;
+    }
+    console.log(result);
+    }
```

Comment:
please correct indentations!

also if...else is not required here you can just use [strict equal operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)

```
function isLetterA(letter) {
  if (letter === "A") {
    return true;
  } else {
    return false;
  }
}

function isLetterB(letter) {
  if (letter === "B") {
    return true;
  }
  return false;
}

function isLetterC(letter) {
  return letter === "C";
}
```
also consider if "NATASZA"/"natasza"/"NaTaSzA" should return true or false? In my opinion this functions `should be` case insensitive so name can be writen even in strage casing and still pass :)